x-pack/plugins/ml/public/explorer/explorer_charts/explorer_charts_container.js

```diff
@@ -64,7 +70,15 @@ function ExplorerChartContainer({
     if (typeof byField !== 'undefined') {
       DetectorLabel = (
         <React.Fragment>
-          {detectorLabel}<br />y-axis event distribution split by &quot;{byField.fieldName}&quot;
+          <FormattedMessage
+            id="xpack.ml.explorer.charts.detectorLabel"
+            defaultMessage="{detectorLabel}{br}y-axis event distribution split by &quot;{fieldName}&quot;"
+            values={{
+              detectorLabel,
+              br: '<br />',
```

Comment:
The `<br />` tag gets escaped and not rendered as HTML:

![image](https://user-images.githubusercontent.com/230104/50635616-32c53d00-0f53-11e9-887d-bcabe05f563b.png)